extension/controller/experiments/TranslationBar/api.js

```diff
@@ -38,10 +75,6 @@ const translationNotificationManagers = new Map();
       const { ExtensionUtils } = ChromeUtils.import("resource://gre/modules/ExtensionUtils.jsm");
       const { ExtensionError } = ExtensionUtils;
 
-      const { Services } = ChromeUtils.import("resource://gre/modules/Services.jsm");
-
-      const { ExtensionCommon } = ChromeUtils.import("resource://gre/modules/ExtensionCommon.jsm");
```

Comment:
I've removed ExtensionCommon because it is unnecessary due to it being part of the environment by default: https://searchfox.org/mozilla-central/rev/ea1234192518e01694a88eac8ff090e4cadf5ca4/toolkit/components/extensions/ExtensionCommon.jsm#1809,1818